lib/streamlit/DeltaGenerator.py

```diff
@@ -44,6 +45,8 @@
 
 MAX_DELTA_BYTES = 14 * 1024 * 1024  # 14MB
 
+DELTAS_WHICH_USE_DATAFRAME = ('line_chart', 'area_chart', 'bar_chart')
```

Comment:
Add comment explaining this. Something like: List of Streamlit commands that perform a Pandas "melt" operation on input dataframes.

Also, the name of this variable isn't exactly correct. There are plenty of deltas that use dataframes, like `st.dataframe`, `st.altair_chart`, `st.map`, etc.

What you mean here is probably `DELTAS_TYPES_THAT_MELT_DATAFRAMES`.

Reply:
Will do, thanks